cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -577,8 +577,13 @@ public Integer call() throws Exception {
   }
 
   protected void deleteArtifact(Id.Artifact artifact, int expectedResponseCode) throws Exception {
+    deleteArtifact(artifact, expectedResponseCode, artifact.getNamespace());
+  }
+
+  protected void deleteArtifact(Id.Artifact artifact, int expectedResponseCode, Id.Namespace namespace)
```

Comment:
Shouldn't introduce this method. If you want to delete an artifact in a given namespace, constructs an `Id.Artifact` in the desired namespace when calling the `deleteArtifact` method.